tools/topology/sof/pipe-volume-demux-playback.m4

```diff
@@ -0,0 +1,124 @@
+# Demux Pipeline
+#
+#  Low Latency Playback with demux.
+#
+# Pipeline Endpoints for connection are :-
+#
+#	Playback Demux
+#	B1 (DAI buffer)
+#
+#
+#  host PCM_P -- B0 --> Demux(M) -- B1--> sink DAI0
```

Comment:
@juimonen missing volume here?

Reply:
@juimonen I think @ranj063 means in the comment, it does not align with PR title as there is no volume.

Reply:
@ranjani @lgirdwood yeah comments are missing the volume component. This is quite direct copy of mixer pipeline, so previously we didn't have any volume at all, this introduces 2 pieces, mainly for format conversion. I'm not sure how demux handles formats if you define it in binary. I wanted to have operations in the middle of the pipe to be s32 and conversions around it. I'm not sure how to do it otherwise?